ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14-nightly-x86.yaml

```diff
@@ -41,7 +41,7 @@ tests:
     workflow: openshift-qe-installer-aws
   timeout: 7h0m0s
 - as: control-plane-24nodes
-  cron: 0 12 2,16 * *
+  cron: 0 8 23 * *
```

Comment:
Oh, because this is another one at hour 8.

If you pick hour=17, we don't have any jobs at that time.

Reply:
Done.